### ISSUE TITLE
Invalid syntax in index.d.ts (v2.3.0)

### ISSUE THREAD

**Issue by pbeyeler** [JustFly1984/react-google-maps-api#2861]
The latest version (2.3.0) that was released just an hour ago seems to be corrupt. 

```
../../node_modules/@react-google-maps/api/dist/index.d.ts(122,24): error TS1005: ')' expected.
../../node_modules/@react-google-maps/api/dist/index.d.ts(122,25): error TS1002: Unterminated string literal.
../../node_modules/@react-google-maps/api/dist/index.d.ts(124,9): error TS1005: ';' expected.
../../node_modules/@react-google-maps/api/dist/index.d.ts(124,22): error TS1005: ';' expected.
../../node_modules/@react-google-maps/api/dist/index.d.ts(124,34): error TS1109: Expression expected.
../../node_modules/@react-google-maps/api/dist/index.d.ts(124,59): error TS1005: ';' expected.
../../node_modules/@react-google-maps/api/dist/index.d.ts(124,68): error TS1002: Unterminated string literal.
```
![image](https://user-images.githubusercontent.com/35919999/133998471-64ee9ced-6dca-4386-80fc-540be3e83587.png)

**Comment by thekevinbrown** [comment]
Thanks for letting us know! I'll have a look at this ASAP, planning to move to `@rollup/plugin-typescript` instead of `rollup-plugin-typescript`.

**Comment by JustFly1984** [comment]
I will merge his PR asap

**Comment by thekevinbrown** [comment]
PR is waiting for review: #2863

**Comment by JustFly1984** [comment]
I've released 2.4.0, and also updated @types/googlemaps to @types/google.maps
please test
